data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
```

Comment:
So what are our exact decommission dates?

BTW, is donation link information preserved? How much who wants to donate to who? Do we have backups of this data? This may be critical to preserve while payments are paused.